proto/cosmos/base/store/internal/kv/v1beta1/kv.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+package cosmos.base.store.internal.kv.v1beta1;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/store/internal/kv";
+
+// This is duplicated from the base kv directory to avoid a circular dependency with the cosmos-sdk
```

Comment:
Why don't we simply change the go_package in the existing proto file to `internal`. Seems weird to have 2 exact same proto files

Reply:
Does this make sense @tac0turtle?

Reply:
I can explore reducing the file but the other kv dependency is used for simulation testing so we can't make it private

Reply:
I was able to remove the kv proto file in types/kv since it's not needed there.

Reply:
cleaned up and now there is only one version of the proto file